.circleci/config.yml

```diff
@@ -35,3 +82,37 @@ workflows:
           context: CodacyAWS
           requires:
             - codacy/publish_docker
+
+  update_vulnerability_dbs:
+    triggers:
+      - schedule:
+          cron: "0 0,12 * * *"
+          # cron: "* * * * *"
+          filters:
+            branches:
+              ignore:
+                - master
+    jobs:
+      - codacy/checkout_and_version
+      - build_and_publish_docker:
+          name: publish_docker_local
+          requires:
+            - codacy/checkout_and_version
+      - install_trivy_and_download_dbs:
+          name: install_trivy_and_download_dbs
+          requires:
+            - publish_docker_local
```

Comment:
```suggestion
      - install_trivy_and_download_dbs:
          name: install_trivy_and_download_dbs
          requires:
            - codacy/checkout_and_version
      - build_and_publish_docker:
          name: publish_docker_local
          requires:
            - install_trivy_and_download_dbs
      
```

It needs to be the other way around. First download the images to a well known directory, then copy the contents of the directory to the Docker image when building